lib/components/Editor/CustomExtensions/Title/ExtensionConfig.js

```diff
@@ -0,0 +1,32 @@
+import Heading from "@tiptap/extension-heading";
+import { mergeAttributes } from "@tiptap/core";
+
+export default Heading.extend({
+  name: "title",
+  defining: true,
+
+  addAttributes() {
+    return {
+      level: {
```

Comment:
Do we really need to add the attributes here explicitly? It's already given as default in the Heading extension. I think we won't need to extend this addAttributes method again unless we have any customization. Please share your thoughts